Web Hacks/simple counter/script.js

```diff
@@ -0,0 +1,16 @@
+const counter = document.getElementById('counter');
+const incr = document.querySelector('.incr');
+const decr = document.querySelector('.decr');
+
+let count = 0;
+incr.addEventListener("click", () => {
+    count++;
+    counter.innerHTML = count;
+})
+decr.addEventListener("click", () => {
+    if(count<=0){
```

Comment:
This is a wonderful beginner project. The CSS looks so cool @dth99 

I have a doubt here. On clicking decrement, if the number is equal less than or equal to 0, why do we need to increase the value?  If you don't want to implement negative numbers then you could keep it equals to 0. 

Just let me know about this, I will merge soon after it.

Reply:
Yes sir, you are right.

Reply:
Sorry I couldnt get you. Why do we need to increase the value of count?

Reply:
Sir, please check now  logic is correct.

Reply:
Hey bro, this is all I meant to say. 

`if(count<=0){
	count=0;
}
else
	count--;
`

Reply:
I get it, you mean only for +ve value.
Now check big bro